### DISCUSSION TITLE
how to upload multiple media ?

### DISCUSSION THREAD

**Discussion by imknight** [awcodes/filament-curator#280] [Q&A]
the create buttonc an only upload one by one, is there anyway to batch upload?

**Comment by lotarbo** [comment]
i do it inside field, here you can upload multiple files
<img width="424" alt="image" src="https://github.com/awcodes/filament-curator/assets/1766890/ff25e156-41b7-43ff-83d4-7c34d82080b9">
not very convenient, it would be cool if the manager could also

**Reply by awcodes** [reply]
What are you referring to with "manager"?

**Reply by lotarbo** [reply]
@awcodes media page)
<img width="1328" alt="image" src="https://github.com/awcodes/filament-curator/assets/1766890/577cf859-7199-4750-91d9-b3ae766e9de1">

**Reply by awcodes** [reply]
There's no way to do multiple there. That is a standard Filament resource. Each media is a separate record in a resource.

And even if you could do multiple upload there, you'd still have to go to each one individually to edit things like alt text, etc.
